src/index.ejs

```diff
@@ -52,6 +52,45 @@
         sentryReleaseVersion: '{!= sentryReleaseVersion !}',
         sentryDebug: {!= sentryDebug !},
         sentrySampleRate: {!= sentrySampleRate !},
+        info: {
+          general: {
+            provider: '{!= info.general.provider !}',
+            installationName: '{!= info.general.installationName !}',
+            availabilityZones: {
+              default: {!= info.general.availabilityZones.default !},
+              max: {!= info.general.availabilityZones.zones.split(' ').filter(Boolean).length !},
+              zones: [
+                {! info.general.availabilityZones.zones.split(' ').filter(Boolean).map((zone) => { !}
+                  '{!= zone !}',
+                {! }) !}
```

Comment:
Here's where the weird stuff starts. Deserializing values.

Not worried about performance since this code is not executed at run time.